cypress/fixtures/docs/staticJSONObjects.md

```diff
@@ -0,0 +1,144 @@
+# Static JSON Objects
+
+Firebolt Calls has two types of JSON objects: Firebolt Calls and Validation Objects. These objects are used to make API calls and validate the response received from the API.
+
+## [Supported Static Glue Codes](../../support/step_definitions/fireboltCalls.md)
+
+## Firebolt Calls
+
```

Comment:
Firebolt JSON objects represent a firebolt api call to be made. It can contain the following parameters:
- method - Firebolt method name in the format module.method. 
- params - Represent the parameters to be sent for the method
- context - Represent the data that needs to be stored in the API/event object that helps to search for a specific object which contains the response in a global list
- expected - Determine whether expecting for an error or result

you can add this as a table. It can contain following columns
1. Field 
2. Required - Y/N
3. Accepted values

Reply:
done